packages/server/api/src/app/workers/helper/redis-rate-limiter.ts

```diff
@@ -0,0 +1,96 @@
+import { JobType, triggerTimeoutSandbox } from '@activepieces/server-shared'
+import { isNil } from '@activepieces/shared'
+import { DefaultJobOptions, Job, Queue, Worker } from 'bullmq'
+import dayjs from 'dayjs'
+
+import { createRedisClient } from '../../database/redis-connection'
+import { AddParams, JOB_PRIORITY } from '../queue/queue-manager'
+import { redisHandler } from '../redis/redis-handler'
+
+
+const EIGHT_MINUTES_IN_MILLISECONDS = 8 * 60 * 1000
+const defaultJobOptions: DefaultJobOptions = {
+    attempts: 5,
+    backoff: {
+        type: 'exponential',
+        delay: EIGHT_MINUTES_IN_MILLISECONDS,
+    },
+    removeOnComplete: true,
+}
+
+
+let worker: Worker | null = null
+
+let queue: Queue | null = null
+
+
+export const redisRateLimiter = {
```

Comment:
This is should be merged with redis-Handler, most of the functions should be internal not inside export const

I think what you need to expose three functions as they are only one used inside redis-queue
- shouldRateLimit
- rateLimitJob is (which is now called delayJob)
- onJobCompleteOrFail